app/src/main/java/com/dgaf/happyhour/Controller/MainActivity.java

```diff
@@ -88,38 +112,61 @@ public void onDrawerOpened(View drawerView) {
         getSupportActionBar().setHomeAsUpIndicator(R.drawable.ic_drawer);
     }
 
-    /*TODO this needs to be declared in its own file*/
-    private class DrawerItemClickListener implements ListView.OnItemClickListener {
-        @Override
-        public void onItemClick(AdapterView parent, View view, int position, long id) {
-            selectItem(position);
-        }
-    }
-
-
     /** Swaps fragments in the main content view */
-    public void selectItem(int position) {
+    private void selectItem(int position) {
 
         Fragment fragment = null;
         String identifier = null;
         /*we will open all the various fragments from the sliding drawer here*/
         switch(position){
+            // List of deals
             case 0:
                 fragment = new ViewPagerFragment();
                 break;
-            case 1:
 
+            // bar
```

Comment:
We should refer to this as "divider" instead of "bar" to keep it consistent across the app.